src/scripts/visualizations/legend.js

```diff
@@ -5,6 +5,7 @@
             vAlign: 'middle',
             hAlign: 'right',
             direction: 'vertical',
+            enabled: true,
```

Comment:
nvm, thought contour already had a `legend: { enabled` property, didn't realize it was just added in tooltip.